design.html

```diff
@@ -0,0 +1,1144 @@
+                     FDR</figcaption>
+         </figure>
+         <figure class="col-md-6 col-lg-3">
+           <img class="w-100" src="./assets/things/bottom14.PNG" alt="politics">
+           <figcaption  class="  h5 pt-3 mr-4 text-justify text-black  font-weight-bold" > Biden, the Moderate, Plans the Most Radical Economic Overhaul Since
+                   FDR</figcaption>
+       </figure>
+     </div>
+      </section>
+      <section class="container-fluid"> <h3>twm</h3>
+        <div class="us-section-tittle " ><a href="#" >U.S</a></div>
+    <section class="d-lg-flex d-md-flex border-bottom  us-section  "> <h3>twm</h3>
+    
+     <figure class="col-md-4 mt-3 col-4 " >
+       <img class=" us-section-bigimg  " src="./assets/things/bottom15.PNG" >
+       <figcaption class="font-weight-bold us-section-bigtxt justify-content h4 text-black ">Don Jr. Set His Sights on Big Tech Months Before His Father's Twitter Feud</figcaption>
+     </figure>
+     <figure  class="col-4 col-md-4 mt-3">
+      <img class=" us-section-bigimg " src="./assets/things/bottom16.PNG" >
+      <figcaption class="font-weight-bold h4  us-section-bigtxt  text-black ">Don Jr. Set His Sights on Big Tech Months Before His Father's Twitter Feud</figcaption>
+    </figure>
+    <div class="   ">
+     <div class="secondary-item ">
+        <div class="aside-pictures  d-inline float-left  ">
+          <div class="container py-3 border-bottom">
+            <div class="media  text-justify">
+              <img class="us-section-img align-self-center  m-0" src="./assets/things/bottom17.PNG"  alt="John Doe" >
+              <div class="media-body ml-md-2">
+                <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                     
+              </div>
+            </div>
+          </div>
+      
+    
+          <div class="container py- border-bottom">
+            <div class="media  text-justify">
+              <img class="us-section-img align-self-center mr-3"  src="./assets/things/bottom18.PNG" alt="John Doe" >
+              <div class="media-body ml-md-2">
+                <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                     
+              </div>
+            </div>
+          </div> <div class="container py-3 border-bottom">
+            <div class="media  text-justify">
+              <img class="us-section-img align-self-center mr-3"  src="./assets/things/bottom19.PNG" alt="John Doe" >
+              <div class="media-body ml-md-2">
+                <a class=" h5 font-weight-bold text-body " href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                     
+              </div>
+            </div>
+          </div>
+    </div>
+    </div>
+    </div>
+    </section>
+
+    <span class="col-12 "> <a href="#" ></a>U.S</span>
```

Comment:
- [ ] The styling of the titles in your page is with the wrong color and size (in many other sections beyond this)